src/functions/components/FunctionRunsRow.tsx

```diff
@@ -15,28 +14,27 @@ import FunctionRunLogsOverlay from 'src/functions/components/FunctionRunLogsOver
 // Types
 import {FunctionRun} from 'src/client/managedFunctionsRoutes'
 import {DEFAULT_TIME_FORMAT} from 'src/shared/constants'
+import {FormattedDateTime} from '../../utils/datetime/FormattedDateTime'
+
+// Utils
```

Comment:
i think this import needs to go under this comment.

also, reminder to import from `src` not from this directory.

```ts
import {FormattedDateTime} from 'src/utils/datetime/FormattedDateTime'

Reply:
Yes, my bad. Just fixed it!